workflow/envs/pystats.yaml

```diff
@@ -6,4 +6,4 @@ dependencies:
   - pyreadr =0.5
   - altair =5.3
   - pyarrow =16.1
-  - vl-convert-python=1.7
+  - vl-convert-python =1.7
```

Comment:
Just for info: the syntax convention for specifying version numbers in conda `.yaml` files always has a space before the range specifiers. I couldn't find a good source for the motivation behind this, but here's an example with the `six` version specification:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run